fcos/release-checklist.md

```diff
@@ -1,5 +1,10 @@
 ---
-# Template generated by https://github.com/coreos/repo-templates; do not edit downstream
+name: {{ stream }} release checklist
```

Comment:
the yaml front matter doesn't work with the `{{ stream }}` placeholder. The file should look like this after it's been rendered: https://github.com/marmijo/fedora-coreos-streams/blob/issue-template-fix-demo/.github/ISSUE_TEMPLATE/stable.md